### ISSUE TITLE
utils.utils & NuCLS data format

### ISSUE THREAD

**Issue by MoriartyG3** [WinnieLaugh/Dataset_Merger#3]
Hello! i could not find the utils.utils script.  Also NuCLS data available on their site does not have .mat labels, could your share the NuCLS data that you used.

**Comment by WinnieLaugh** [comment]
Thanks a lot for pointing this out! 

I just checked the code and found that you were right. I will take a look at this tomorrow and upload the scipt and the data when I get back to school.

**Comment by WinnieLaugh** [comment]
Hi @1AlgoG3 

1. For the .mat labels, yes, NuCLS does not have those labels, I inferred them with a trained hovernet (retrained with all segmentation datasets) as the original labels are not so clean.

2. For the utils.utils script, I am so sorry that I could not find it. I have just finished my Ph.D's study and got into a new school. I think I must have lost it when I got into chaos copying the data from my original server. But still, only three functions in the utils.utils are used and I think they are:

- The 'draw_dilation' function should be the same as the models/utils.py
- The 'img_mask_alignment' aligns the inferred masks with the images, as some of the inferring may lead to wrong image sizes (typically +-1 for the original sizes, there are not many of them), as I recalled.
- The 'img_mask_alignment' function is not used and could be deleted.
